pkg/adapter/metrics.go

```diff
@@ -27,11 +30,12 @@ const (
 
 // Label kinds supported by mixer.
 const (
-	String LabelKind = iota
+	String LabelType = iota
```

Comment:
Attributes are *used* in the API layer, but their definitions have wider use.
They are used in the policy language, which is orthogonal to the API.

The expression language is only aware of config.Value_Types, so we will perpetually have to maintain this mapping.